cobra/core/metabolite.py

```diff
@@ -216,30 +216,40 @@ def summary(self, solution=None, threshold=0.01, fva=None, names=False,
 
         Parameters
         ----------
-        solution : cobra.Solution, optional
+        solution: cobra.Solution or None, optional (default None)
```

Comment:
> Also, the default should be provided in the description.

The documentation also states that: _Optional keyword parameters have default values, which are displayed as part of the function signature._ . This is what led me to write it in that fashion. Also, the style is not consistent across cobrapy itself. I feel we should make it consistent to avoid future issues. So, do we stick to what you propose? I feel stating the default in the function signature makes it a bit concise.

Reply:
Yes, cobrapy is a patchwork :) Since we decided to use numpydoc, I want like to closely follow it. So whenever we update or otherwise edit a part of cobrapy, we should improve the docstrings where necessary.